tests/test_io_elementwise.py

```diff
@@ -41,6 +41,16 @@
     G = TypeVar("G", H5Group, ZarrGroup)
 
 
+pytestmark = [
+    pytest.mark.filterwarnings(
+        "default:.*Structured:zarr.core.dtype.common.UnstableSpecificationWarning"
+    ),
+    pytest.mark.filterwarnings(
+        "default:.*FixedLengthUTF32:zarr.core.dtype.common.UnstableSpecificationWarning"
+    ),
+]
```

Comment:
I would guess that this is somehow the source of the issue given the stack trace.  But why that is...